lib/internal/worker.js

```diff
@@ -212,9 +212,14 @@ class Worker extends EventEmitter {
     }
 
     if (!options.eval) {
-      if (!path.isAbsolute(filename)) {
-        throw new ERR_WORKER_NEED_ABSOLUTE_PATH(filename);
+      if (!path.isAbsolute(filename) &&
+        !filename.startsWith('./') &&
+        !filename.startsWith('../')) {
+
```

Comment:
I think indentation is off here?

Reply:
Lint didn't seem to have a problem with it, so not sure what should be different...

Reply:
We usually line up the individual clauses vertically (and omit a blank link at the beginning of a block) :)